CodeSniffer/Standards/Generic/Sniffs/WhiteSpace/DisallowSpaceIndentSniff.php

```diff
@@ -94,42 +94,60 @@ public function process(PHP_CodeSniffer_File $phpcsFile, $stackPtr)
                 continue;
             }
 
-            // If tabs are being converted to spaces, the original content
+            // If spaces are being converted to tabs, the original content
```

Comment:
I've reverted the word order, but adjusted the comment slightly for clarification of what was meant here.